components/automate-ui/src/app/pages/policy/details/policy-details.component.html

```diff
@@ -36,15 +36,15 @@
       </section>
     </section>
     <section class="page-body" *ngIf="tabValue === 'members'">
-      <div *ngIf="(members$ | async)?.length === 0" class="empty-state">
+      <div *ngIf="members.length === 0" class="empty-state">
```

Comment:
I've been wondering if it's better to use `members` or observable of `members` in the template. Why did you change it here?

Reply:
In all the other places I've seen we just initialize to an empty array and then populate after the subscribe finishes. Doesn't look to me that the complexity of having it be an obserable gives us anything? Switching it to a plain old array just simplified the templates and made the ts code more standardized.